gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -19,6 +19,12 @@
         <script src="../../../../ontologyHandler/src/ontologyHandlerUtilities.js"></script>
         <script src="../../../src/CanopyMatchMakerUtilities.js"></script>
         <script src="../js/CanopyMatchMakerUtilitiesTests.js"></script>
+
+        <!-- This is required not for the tests themselves, but for the coverage client -->
+        <script src="../../../../../../node_modules/infusion/src/framework/core/js/FluidDocument.js"></script>
```

Comment:
Confirmed, and also found two more dependencies that are not needed (see below).